core/src/main/scala/org/apache/spark/scheduler/cluster/mesos/package.scala

```diff
@@ -0,0 +1,96 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.spark.scheduler.cluster
+
+import java.util.{List => JList}
+
+import com.google.common.base.Splitter
+import org.apache.mesos.Protos
+import org.apache.mesos.Protos._
+
+import scala.collection.JavaConversions._
+
+
+package object mesos {
```

Comment:
Yeah that's a good point. I think the convention elsewhere is that we define a `XUtils` object and do `XUtils.methodName()` for common methods (see `Utils`, `JettyUtils`, `AkkaUtils` etc.). It might make more sense to do that here.